System.IO.Abstractions/FileWrapper.cs

```diff
@@ -214,6 +263,18 @@ public override string ReadAllText(string path, Encoding encoding)
             return File.ReadAllText(path, encoding);
         }
 
+#if NETCOREAPP2_0
+        public override async Task<string> ReadAllTextAsync(string path, CancellationToken cancellationToken)
+        {
+            return await File.ReadAllTextAsync(path, cancellationToken);
```

Comment:
@robkeim Should I leave the Read methods as is with `async/await`, or is there a difficult way you'd prefer for me to handle them?

Reply:
In this case you can remove the `async` and `await` keywords and it will compile since you're returning an overload of `ReadAllTextAsync` which also returns the same type `Task<string>`

Reply:
Done.